### PR TITLE
Completed home page

### PR DESCRIPTION
**Pull request by mohitdhatrak**
Deploy preview: https://deploy-preview-2--onsport.netlify.app/

Change log: Added home page with different page routes

Screenshots: 

image

image

### CODE REVIEW

File: src/components/CategoryCard.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+
+export function CategoryCard({ category }) {
```
Comment (VishakAmin) [inline src/components/CategoryCard.jsx]: category can be destructed more here
`{ category: { imageSrc, text, subText}}`

File: src/components/TopNavigation.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import logo from "../assets/website-logo.png";
+import { Link } from "react-router-dom";
+import "../styles/TopNavigation.css";
+
```
Comment (VishakAmin) [inline src/components/TopNavigation.jsx]: If you separate the module import with local import will increase the readability
Ex:
```
//All module imports at the start
import React from "react";

//All local import
import "./landingPage.css"
```

File: src/pages/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                            >
+                                Shop now
+                            </Link>
+                        </div>
+                    </div>
+                </section>
+
+                <section className="cards-container">
+                    {productCategories.map((category, index) => (
+                        <CategoryCard key={index} category={category} />
```
Comment (VishakAmin) [inline src/pages/Home.jsx]: Avoid using index as **_key_**. You can have a look [here](https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66)  for the reason why we shouldn’t use.

**Review by VishakAmin — COMMENTED** [review]
Left few comments to have alook.

File: src/pages/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    shoesBanner,
+    trophiesBanner,
+} from "../assets/home";
+import { Link } from "react-router-dom";
+import "../styles/Home.css";
+import { TopNavigation } from "../components/TopNavigation.jsx";
+import { CategoryCard } from "../components/CategoryCard.jsx";
+
+export function Home() {
+    const productCategories = [
```
Comment (irajbhinde) [inline src/pages/Home.jsx]: Looks good to me. Just add the following objects & import the images at category.js when you integrate mockbee in your project. Category.js would be generated by mockbee itself

**Review by irajbhinde — COMMENTED** [review]
Looks fine! Would suggest to Integrate mockbee as soon as possible to avoid further inconvenience

File: src/components/CategoryCard.jsx (hunk 1 above)
Comment (mohitdhatrak) [reply src/components/CategoryCard.jsx]: Done

File: src/components/TopNavigation.jsx (hunk 2 above)
Comment (mohitdhatrak) [reply src/components/TopNavigation.jsx]: Thanks, nice suggestion!

Comment (mohitdhatrak) [reply src/components/TopNavigation.jsx]: Done

File: src/pages/Home.jsx (hunk 3 above)
Comment (mohitdhatrak) [reply src/pages/Home.jsx]: Thanks! Glad you shared

File: src/pages/Home.jsx (hunk 4 above)
Comment (mohitdhatrak) [reply src/pages/Home.jsx]: Sure, did that now

**Pull request merged**